client/src/features/Event/Overview/components/NicknameModal.tsx

```diff
@@ -0,0 +1,110 @@
+import { useState } from 'react';
+
+import { css } from '@emotion/react';
+import { useNavigate } from 'react-router-dom';
+
+import { useCreateProfile } from '@/api/mutations/useCreateProfile';
+import { Button } from '@/shared/components/Button';
+import { Flex } from '@/shared/components/Flex';
+import { Icon } from '@/shared/components/Icon';
+import { Input } from '@/shared/components/Input';
+import { Modal } from '@/shared/components/Modal/Modal';
+import { Text } from '@/shared/components/Text';
+
+type NicknameModalProps = {
+  isOpen: boolean;
+  onClose: () => void;
+};
+
+export const NicknameModal = ({ isOpen, onClose }: NicknameModalProps) => {
+  //E.TODO 추후 organizationId 받아오기
+  const createProfileMutation = useCreateProfile(1);
+  const navigate = useNavigate();
+  const [nickname, setNickname] = useState('');
+
+  const handleNicknameSubmit = (nickname: string) => {
+    createProfileMutation.mutate(
+      { nickname },
+      {
+        onSuccess: () => {
+          alert('조직 참가가 완료되었습니다!');
+          navigate('/event');
+          onClose();
+        },
+        onError: () => {
+          alert('참가 중 오류가 발생했습니다. 다시 시도해 주세요.');
+        },
+      }
+    );
+  };
+
+  const handleSubmit = () => {
+    if (nickname.trim()) {
+      handleNicknameSubmit(nickname.trim());
+    }
+  };
+
+  const handleClose = () => {
+    setNickname('');
+    onClose();
+  };
+
+  return (
+    <Modal
+      isOpen={isOpen}
+      onClose={handleClose}
+      showCloseButton={false}
+      css={{ maxWidth: '360px' }}
+    >
+      <Flex dir="column" gap="20px">
+        <Flex justifyContent="space-between" alignItems="flex-start">
+          <Text type="Body" weight="bold" color="#333">
+            닉네임 설정
+          </Text>
+          <Icon name="close" onClick={handleClose} />
```

Comment:
p1.
시맨틱 측면에서 봤을 때, IconButton 컴포넌트를 사용하는 게 더 적절해 보여요!

```suggestion
          <IconButton name="close" onClick={handleClose} />
```